### ISSUE TITLE
ValueError on calling R_hfs with a given t

### ISSUE THREAD

**Issue by elitherl** [ukaea/Indica#39]
**Describe the bug**
Calling `Equilibrium.R_hfs` with given values for both `rho` and `t` results in a call to `R = rmji.interp(rho_poloidal=rho, method="cubic") - self.R_offset`. As both `rho` and `rmji` have a common coordinate `t`, `interp` attempts 2D interpolation and `method="cubic"`

**Comment by elitherl** [comment]
```python
        #  Snippet from equilibrium.py
        if t is None:
            rmji = self.rmji
            t = self.rmji.coords["t"]
            rho, _ = self.convert_flux_coords(rho, t, kind, "poloidal")
            R = rmji.indica.interp2d(rho_poloidal=rho, method="cubic") - self.R_offset
        else:
            rmji = self.rmji.interp(t=t, method="nearest")
            rho, _ = self.convert_flux_coords(rho, t, kind, "poloidal")
            R = rmji.interp(rho_poloidal=rho, method="cubic") - self.R_offset
```
If `t` isn't given to `R_hfs`, `.indica.interp2d` is used instead. Changing the case where `t` is given to also use this solves the issue

**Comment by elitherl** [comment]
Solved by #40
